src/rules/declaration-property-value-no-unknown/README.md

```diff
@@ -0,0 +1,68 @@
+# declaration-nested-properties-no-divided-groups
+
+Disallow nested properties of the same "namespace" be divided into multiple groups.
```

Comment:
This seems to be the wrong README file.

Reply:
Huh, I do not know how that got there. 🤔